Development/src/pages/receivers/ReceiversShow.js

```diff
@@ -83,29 +93,28 @@ const ReceiversShow = props => {
                             component={Link}
                             to={`${props.basePath}/${props.id}/show/`}
                         />
-                        {get(controllerProps.record, '$connectionAPI') !==
-                            undefined &&
-                            ['active', 'staged'].map(label => (
-                                <Tab
-                                    key={label}
-                                    label={label}
-                                    value={`${props.match.url}/${label}`}
-                                    component={Link}
-                                    to={`${props.basePath}/${props.id}/show/${label}`}
-                                    disabled={
-                                        !get(
-                                            controllerProps.record,
-                                            `$${label}`
-                                        )
-                                    }
-                                />
-                            ))}
+                        {['active', 'staged'].map(label => (
+                            <Tab
+                                key={label}
+                                label={label}
+                                value={`${props.match.url}/${label}`}
+                                component={Link}
+                                to={`${props.basePath}/${props.id}/show/${label}`}
+                                disabled={
+                                    !get(controllerProps.record, `$${label}`) ||
+                                    !useConnectionAPI
+                                }
+                            />
+                        ))}
                         <Tab
                             label="Connect"
                             value={`${props.match.url}/connect`}
                             component={Link}
                             to={`${props.basePath}/${props.id}/show/connect`}
-                            disabled={!get(controllerProps.record, '$active')}
+                            disabled={
```

Comment:
Can't see that it matters either way, but I think using `'$staged'` makes more sense logically cos that's what the connect tab uses, isn't it?